### PR TITLE
Partial Mobile Support

### PR DESCRIPTION
**Pull request by Geometrically**
This PR adds partial mobile support, with some missing pages related to mod management, which aren't really useful for the typical mobile user.

These pages include:
Mod Create/Edit Page
Version Create/Edit Page
Mod Settings Page (Partial Support, but teams management interface is glitchy)

All other pages work and look fine on mobile!

### CODE REVIEW

**Review by Mysterious-Dev — APPROVED** [review]
It's good in my test.

**Review by falseresync — CHANGES_REQUESTED** [review]
Close navabar on route change
Use `watch` hook for that: https://stackoverflow.com/questions/42855985/vuejs-close-hamburger-menu-on-route-change

File: components/ui/ProjectCard.vue [hunk 1]
```diff
@@ -1,103 +1,109 @@
...
-        <p v-if="author" class="author">
-          by <nuxt-link :to="'/user/' + author">{{ author }}</nuxt-link>
-        </p>
+    <div class="columns">
+      <div class="icon">
+        <nuxt-link v-if="isModrinth" :to="'/mod/' + id">
+          <img
+            :src="
+              iconUrl
+                ? iconUrl
```
Comment (rubybb) [inline components/ui/ProjectCard.vue:9]: Use the ``Logical OR (||)`` operator.

File: components/ui/ProjectCard.vue [hunk 2]
```diff
@@ -1,103 +1,109 @@
...
+        </div>
+        <p class="description">
+          {{ description }}
+        </p>
+        <div :class="{ vertical: editMode }" class="bottom">
+          <div class="stats">
+            <div v-if="status !== null" class="stat">
+              <div class="info">
+                <h4>Status</h4>
+                <span v-if="status === 'approved'" class="badge green">
```
Comment (rubybb) [inline components/ui/ProjectCard.vue:36]: Does vue have switch cases?

Comment (falseresync) [reply components/ui/ProjectCard.vue:36]: No

File: components/ui/ProjectCard.vue (hunk 1 above)
Comment (Redblueflame) [reply components/ui/ProjectCard.vue:9]: Or the ES6 ??, more stable and not relying on some js trickery

Comment (rubybb) [reply components/ui/ProjectCard.vue:9]: > Or the ES6 ??, more stable and not relying on some js trickery

In this case it'd be better to use ``||`` as values like an empty string should just default to the placeholder icon, with ``??`` it  would not.

**Pull request merged**